generators/client/templates/angular/src/main/webapp/app/admin/configuration/configuration.service.ts.ejs

```diff
@@ -17,70 +17,79 @@
  limitations under the License.
 -%>
 import { Injectable } from '@angular/core';
-import { HttpClient, HttpResponse } from '@angular/common/http';
+import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 
 import { SERVER_API_URL } from 'app/app.constants';
 
-@Injectable({providedIn: 'root'})
-export class <%=jhiPrefixCapitalized%>ConfigurationService {
+export interface ConfigProps {
+    contexts: Contexts;
+}
 
-    constructor(private http: HttpClient) {
-    }
+export interface Contexts {
+    [key: string]: Context;
+}
+
+export interface Context {
+    beans: Beans;
+    parentId?: any;
+}
+
+export interface Beans {
+    [key: string]: Bean;
+}
+
+export interface Bean {
+    prefix: string;
+    properties: any;
+}
+
+export interface Env {
+    activeProfiles?: string[];
+    propertySources: PropertySource[];
+}
+
+export interface PropertySource {
+    name: string;
+    properties: Properties;
+}
+
+export interface Properties {
+    [key: string]: Property;
+}
+
+export interface Property {
+    value: string;
+    origin?: string;
+}
+
+@Injectable({providedIn: 'root'})
+export class ConfigurationService {
+    constructor(private http: HttpClient) {}
 
-    get(): Observable<any> {
-        return this.http.get(SERVER_API_URL + 'management/configprops', { observe: 'response' }).pipe(
-            map((res: HttpResponse<any>) => {
-                const properties: any[] = [];
-                const propertiesObject = this.getConfigPropertiesObjects(res.body);
-                for (const key in propertiesObject) {
-                    if (Object.prototype.hasOwnProperty.call(propertiesObject, key)) {
-                        properties.push(propertiesObject[key]);
+    getBeans(): Observable<Beans> {
+        return this.http.get<ConfigProps>(SERVER_API_URL + 'management/configprops').pipe(
+            map(configProps => {
+                // This code is for Spring Boot 2
+                for (const key in configProps.contexts) {
+                    // If the key is not bootstrap, it will be the ApplicationContext Id
+                    // For default app, it is baseName
+                    // For microservice, it is baseName-1
+                    if (!key.startsWith('bootstrap')) {
```

Comment:
Why this was added ? Looks unrelated to the strict option no ?

Reply:
As I removed data conversion from one representation to another then code became so short that I moved functionality from `getConfigPropertiesObjects` to this function and removed `getConfigPropertiesObjects`. So this is not added by this PR but moved from one function to another.

But if I looked at React code now then I see that React is not dropping any context, React is showing all available information. So I'll do the same here, let's show all available information from `configprops` endpoint like this is done in React.

Reply:
Now using all contexts beans as this is in React side.